test.js

```diff
@@ -20,14 +24,14 @@ test('main', async t => {
 		list.every(x =>
 			typeof x.pid === 'number'
 			&& typeof x.name === 'string'
-			&& typeof x.ppid === 'number',
+			&& typeof x.cmd === 'string',
 		),
 	);
 
 	if (!isWindows) {
 		t.true(
 			list.every(x =>
-				typeof x.cmd === 'string'
+				typeof x.ppid === 'number'
 				&& typeof x.cpu === 'number'
```

Comment:
The changes to tests here are because fast-list and task-list support different properties.